src/Navbar.jsx

```diff
@@ -3,14 +3,21 @@ import styled from "styled-components";
 import { Link } from 'react-router-dom';
 
 export const StyledNavbar = styled.nav`
-  text-align: center;
-  background: papayawhip;
-`;
+  background: #322d38;
+  width: 100%;
```

Comment:
If you're working with 100vh for the body height, it make sense to use 100vw here for the width :)